Documentation/release-notes/3855.md

```diff
@@ -0,0 +1,12 @@
+### Java exception wrapping
```

Comment:
...for example, PR #3855 is part of d16-5.  As such, does this file need to exist in master, which will become d16-6?  I don't think so...?